application/templates/cms/measure_actions.html

```diff
@@ -14,9 +14,14 @@
                     {% if measure.latest and measure.status == 'APPROVED' and current_user.can(CREATE_VERSION) %}
                         <a href="{{ url_for('cms.new_version', topic=topic.guid, subtopic=subtopic.guid, measure=measure.guid, version=measure.version) }}" id="measure-action-section__create_new_link-{{ measure.guid }}">Create&nbsp;new&nbsp;version</a>
                     {%  else %}
-                        <a href="{{ url_for('cms.edit_measure_page', topic=topic.guid, subtopic=subtopic.guid, measure=measure.guid, version=measure.version) }}" id="measure-action-section__view_form_link-{{ measure.guid }}">View form</a>
+                        <a href="{{ url_for('cms.edit_measure_page', topic=topic.guid, subtopic=subtopic.guid, measure=measure.guid, version=measure.version) }}" id="measure-action-section__view_form_link-{{ measure.guid }}">View&nbsp;form</a>
                     {% endif %}
                 {% endif %}
+                {% if current_user.can(COPY_MEASURE) %}
```

Comment:
@TheDoubleK I think it'd be a cleaner user interface to have a "Create copy" button on the edit view of the measure itself, rather than on the topic page (as that way you only need to see one button rather than many, you're less likely to click the wrong one, and more likely to check which one you're copying). It'd also avoid the need for the error message on the topic page, as it'd logically appear on the edit form instead.

Reply:
Yep this sounds good.